src/com/tony/server/response/DeleteResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.tony.server.response;
+
+import java.io.File;
+import java.util.HashMap;
+
+public class DeleteResponse extends Response{
+    private final String filePath;
+
+    public DeleteResponse(String filePath) {
+        this.filePath = filePath;
+        setStatusLine("HTTP/1.1 200 OK\n");
+        setHeaders(new HashMap<>());
+        setBody("");
+    }
+
+    public byte[] respond(){
+        new File(filePath).delete();
```

Comment:
It returns a 404 if there's no route to the file. It's supposed to be a 200 if the file to be deleted is already deleted